### PR TITLE
Change driver constructor to fit syntax of latest Python SDK

### PR DESCRIPTION
**Pull request by Rantzur1992**
Changed jobname and projectname values to fit the syntax of the latest Python SDK.

Signed-off-by: Ran Tzur <ran.tzur@testproject.io>

### CODE REVIEW

**Comment by Rantzur1992** [comment]
@vitalybu Besides the change required from the change in out SDK, there seems to be an issue with set timeout of Chrome and Safari latest builds, I had to remove it for the CI to pass.
Also, Evaluate throws an error on the latest Robot build, seems to be a bug on their end.

Had to make this changes for the library to work until they are solved.

File: tests/test_demo_site.robot [hunk 1]
```diff
@@ -22,8 +22,7 @@ Submit Form
 
 *** Keywords ***
 Init
-    ${options}=     Headless Chrome
-    Init Testproject Driver     chrome    timeout=5000      job_name=TestProject Robot  url=https://example.testproject.io/web/     desired_capabilities=${options}
+    Init Testproject Driver     chrome    timeout=30      job_name=TestProject Robot  url=https://example.testproject.io/web/
```
Comment (vitalybu) [inline tests/test_demo_site.robot:25]: So you changed 5000 milliseconds (which is 5 seconds) to 30 seconds?

File: tests/test_demo_site.robot [hunk 2]
```diff
@@ -36,12 +35,6 @@ Clear Fields
     Clear Element Text     css:#name
     Clear Element Text     css:#password
 
-Headless Chrome
-    ${chrome_options} =     Evaluate    sys.modules['selenium.webdriver'].ChromeOptions()    sys, selenium.webdriver
-    Call Method    ${chrome_options}    add_argument    headless
-    Call Method    ${chrome_options}    add_argument    disable-gpu
-    [Return]       ${chrome_options}
-
```
Comment (vitalybu) [inline tests/test_demo_site.robot:44]: Why was this removed?

**Comment by vitalybu** [comment]
> @vitalybu Besides the change required from the change in our SDK, there seems to be an issue with set timeout of Chrome and Safari latest builds, I had to remove it for the CI to pass.
> Also, Evaluate throws an error on the latest Robot build, seems to be a bug on their end.
> 
> Had to make this changes for the library to work until they are solved.

Maybe it's because you changed the timeout units?
Any change that `set_script_timeout` expects milliseconds?
If the tests work (consistently) without it, why was it there in the first place?

File: tests/test_demo_site.robot (hunk 1 above)
Comment (Rantzur1992) [reply tests/test_demo_site.robot:25]: Yes, but it's ignored anyway.

File: tests/test_demo_site.robot (hunk 2 above)
Comment (Rantzur1992) [reply tests/test_demo_site.robot:44]: No longer works on the latest Robot framework version, the Evaluate method does not accept that structure anymore, I tried other ways but none of them worked.

**Comment by Rantzur1992** [comment]
> > @vitalybu Besides the change required from the change in our SDK, there seems to be an issue with set timeout of Chrome and Safari latest builds, I had to remove it for the CI to pass.
> > Also, Evaluate throws an error on the latest Robot build, seems to be a bug on their end.
> > Had to make this changes for the library to work until they are solved.
> 
> Maybe it's because you changed the timeout units?
> Any change that `set_script_timeout` expects milliseconds?
> If the tests work (consistently) without it, why was it there in the first place?

I didn't change anything regarding to the time units, latest build (which was on December) worked fine and it was as MS, now they changed it to seconds, and regardless, it does not work on Chrome and Safari, it throws an excpetion.

File: tests/test_demo_site.robot (hunk 1 above)
Comment (vitalybu) [reply tests/test_demo_site.robot:25]: If ignored, why is it there?

File: tests/test_demo_site.robot (hunk 2 above)
Comment (vitalybu) [reply tests/test_demo_site.robot:44]: Is there another way to pass options?
How should one start a headless session?

File: tests/test_demo_site.robot (hunk 1 above)
Comment (Rantzur1992) [reply tests/test_demo_site.robot:25]: I'll remove it.

**Comment by vitalybu** [comment]
> > > @vitalybu Besides the change required from the change in our SDK, there seems to be an issue with set timeout of Chrome and Safari latest builds, I had to remove it for the CI to pass.
> > > Also, Evaluate throws an error on the latest Robot build, seems to be a bug on their end.
> > > Had to make this changes for the library to work until they are solved.
> > 
> > 
> > Maybe it's because you changed the timeout units?
> > Any change that `set_script_timeout` expects milliseconds?
> > If the tests work (consistently) without it, why was it there in the first place?
> 
> I didn't change anything regarding to the time units, latest build (which was on December) worked fine and it was as MS, now they changed it to seconds, and regardless, it does not work on Chrome and Safari, it throws an excpetion.

What doesn't work? What throws an exception?
Maybe we shouldn't update to the latest Robot version in this case?

File: tests/test_demo_site.robot (hunk 2 above)
Comment (Rantzur1992) [reply tests/test_demo_site.robot:44]: Just pass headless<browsername> instead. in the browser argument

**Comment by Rantzur1992** [comment]
> > > > @vitalybu Besides the change required from the change in our SDK, there seems to be an issue with set timeout of Chrome and Safari latest builds, I had to remove it for the CI to pass.
> > > > Also, Evaluate throws an error on the latest Robot build, seems to be a bug on their end.
> > > > Had to make this changes for the library to work until they are solved.
> > > 
> > > 
> > > Maybe it's because you changed the timeout units?
> > > Any change that `set_script_timeout` expects milliseconds?
> > > If the tests work (consistently) without it, why was it there in the first place?
> > 
> > 
> > I didn't change anything regarding to the time units, latest build (which was on December) worked fine and it was as MS, now they changed it to seconds, and regardless, it does not work on Chrome and Safari, it throws an excpetion.
> 
> What doesn't work? What throws an exception?
> Maybe we shouldn't update to the latest Robot version in this case?

The selenium method in selenium library throws an exception when starting the browsers.
The robot version that I used came with our dependency of selenium library.

File: tests/test_demo_site.robot (hunk 2 above)
Comment (vitalybu) [reply tests/test_demo_site.robot:44]: How do you pass a browser argument to the Agent?
Maybe the "Headless Chrome" example you removed should be replaced with a working alternative?

**Comment by vitalybu** [comment]
> > > > > @vitalybu Besides the change required from the change in our SDK, there seems to be an issue with set timeout of Chrome and Safari latest builds, I had to remove it for the CI to pass.
> > > > > Also, Evaluate throws an error on the latest Robot build, seems to be a bug on their end.
> > > > > Had to make this changes for the library to work until they are solved.
> > > > 
> > > > 
> > > > Maybe it's because you changed the timeout units?
> > > > Any change that `set_script_timeout` expects milliseconds?
> > > > If the tests work (consistently) without it, why was it there in the first place?
> > > 
> > > 
> > > I didn't change anything regarding to the time units, latest build (which was on December) worked fine and it was as MS, now they changed it to seconds, and regardless, it does not work on Chrome and Safari, it throws an excpetion.
> > 
> > 
> > What doesn't work? What throws an exception?
> > Maybe we shouldn't update to the latest Robot version in this case?
> 
> The selenium method in selenium library throws an exception when starting the browsers.
> The robot version that I used came with our dependency of selenium library.

And why do you blame this on the Chrome/Safari containers?
It works without issues with the Desktop versions? It works with older container builds?

File: tests/test_demo_site.robot (hunk 2 above)
Comment (Rantzur1992) [reply tests/test_demo_site.robot]: I'll look for another one, could not find so far.

**Comment by Rantzur1992** [comment]
> > > > > > @vitalybu Besides the change required from the change in our SDK, there seems to be an issue with set timeout of Chrome and Safari latest builds, I had to remove it for the CI to pass.
> > > > > > Also, Evaluate throws an error on the latest Robot build, seems to be a bug on their end.
> > > > > > Had to make this changes for the library to work until they are solved.
> > > > > 
> > > > > 
> > > > > Maybe it's because you changed the timeout units?
> > > > > Any change that `set_script_timeout` expects milliseconds?
> > > > > If the tests work (consistently) without it, why was it there in the first place?
> > > > 
> > > > 
> > > > I didn't change anything regarding to the time units, latest build (which was on December) worked fine and it was as MS, now they changed it to seconds, and regardless, it does not work on Chrome and Safari, it throws an excpetion.
> > > 
> > > 
> > > What doesn't work? What throws an exception?
> > > Maybe we shouldn't update to the latest Robot version in this case?
> > 
> > 
> > The selenium method in selenium library throws an exception when starting the browsers.
> > The robot version that I used came with our dependency of selenium library.
> 
> And why do you blame this on the Chrome/Safari containers?
> It works without issues with the Desktop versions? It works with older container builds?

Never said the issue is with the containers, it happens on Desktop versions as well.

File: README.md [hunk 3]
```diff
@@ -253,7 +253,6 @@ The method is defined with the following arguments:
             self,
             browser: str = "firefox",
             url: Optional[str] = None,
-            timeout: Optional[int] = 5000,
```
Comment (vitalybu) [inline README.md:256]: Is there any other way for a user to control explicit/implicit timeouts?

Comment (Rantzur1992) [reply README.md:256]: Yes, there are keywords for that.
https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Set%20Selenium%20Implicit%20Wait

And a few others.

**Review by vitalybu — APPROVED** [review]
